webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -9819,6 +9851,11 @@ var KM_ATTACHMENT_V2_SUPPORTED_MIME_TYPES = ["application","text","image"];
                                         KommunicateUI.showClosedConversationBanner(true);
                                     }, MCK_BOT_MESSAGE_DELAY);
                                 } else {
+                                    if (kommunicate._globals.isConvJustResolved) {
```

Comment:
can this be replaced with:
kommunicate._globals.isConvJustResolved = !kommunicate._globals.isConvJustResolved